core/textile/sharing.go

```diff
@@ -302,7 +302,7 @@ func (tc *textileClient) buildInvitationSharedDirEntry(
 				SizeInBytes:   strconv.FormatInt(size, 10),
 				FileExtension: ext,
 				Created:       strconv.FormatInt(time.Unix(0, file.CreatedAt).Unix(), 10),
-				Updated:       strconv.FormatInt(time.Unix(updatedAt, 0).Unix(), 10),
+				Updated:       strconv.FormatInt(time.Unix(0, updatedAt).Unix(), 10),
```

Comment:
Should this not be in `RFC3339` format as well?

Reply:
@jsonsivar 

Thank you for spotting this, fixed in 0d4582cefb87735995e6c78e73e251a12a07e624

I relied on the existing convention, eg: https://github.com/FleekHQ/space-daemon/blob/833529cd1e26d96e1914d73adbe5088bc47db0b4/core/textile/sharing.go#L212